app/src/constants.ts

```diff
@@ -195,6 +195,9 @@ export abstract class Constants {
         es_ES: "20210808180117-6v0mkxr",
         ja_JP: "20240530133126-axarxgx",
         it_IT: "20210808180117-6v0mkxr",
+        de_DE: "20241003122725-f93k2y7",
+        he_IL: "20241003122725-f93k2y7",
+        ru_RU: "20241003122725-f93k2y7",
```

Comment:
The added language should point to the English user guide: "20210808180117-6v0mkxr"

Reply:
Keeping the new dates or should I use the original date?